fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MatcherHelper.kt

```diff
@@ -110,9 +110,11 @@ object MatcherHelper {
     fun assertItemWithResIdAndTextExists(vararg appItems: UiObject, exists: Boolean = true) {
         for (appItem in appItems) {
             if (exists) {
-                assertTrue(appItem.waitForExists(waitingTime))
+                assertTrue("${appItem.selector} does not exist", appItem.waitForExists(waitingTime))
+                Log.i(TAG, "assertItemWithResIdExists: Verified ${appItem.selector} exists")
             } else {
-                assertFalse(appItem.waitForExists(waitingTimeShort))
+                assertFalse("${appItem.selector} exists", appItem.waitForExists(waitingTimeShort))
+                Log.i(TAG, "assertItemWithResIdExists: Verified ${appItem.selector} exists")
```

Comment:
Just sanity checking here to double check with your assert messages say versus what is logged afterwards, it's quite confusing e.g `if (exists)` and then the next line says it does not exist.

Reply:
Yes, made a mistake with this one.